src/components/ChatApp/Settings/AccountTab.react.js

```diff
@@ -73,14 +73,19 @@ const AccountTab = props => {
         <Translate text="Select TimeZone" />
       </div>
       <br />
```

Comment:
<br/> not needed, instead use margin in your time-zone class

Reply:
I did not get you.

Reply:
\<br/> tag is not needed. Instead, use margin. The br tag got parsed as html in my comment.